extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -1127,18 +1127,19 @@ static public function addArticlePageToWatchlist( $comment ) {
 	 * @return Bool true -- because it's a hook
 	 */
 	static public function watchlistNotify( RecentChange &$oRC ) {
-		global $wgEnableGroupedArticleCommentsRC;
+		$wg = F::app()->wg;
 
 		Hooks::run( 'AC_RecentChange_Save', [ &$oRC ] );
 
-		if ( !empty( $wgEnableGroupedArticleCommentsRC ) && ( $oRC instanceof RecentChange ) ) {
-			$title = $oRC->getAttribute( 'rc_title' );
-			$namespace = $oRC->getAttribute( 'rc_namespace' );
-			$article_id = $oRC->getAttribute( 'rc_cur_id' );
-			$title = Title::newFromText( $title, $namespace );
+		if ( $wg->EnableGroupedArticleCommentsRC && ( $oRC instanceof RecentChange ) ) {
+			$title = $oRC->getAttribute('rc_title');
+			$namespace = $oRC->getAttribute('rc_namespace');
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "rc_namespace" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)